p2p/node/peerManager/peerManager.go

```diff
@@ -507,7 +487,7 @@ func (pm *BasicPeerManager) UnprotectPeer(peer p2p.PeerID) {
 }
 
 func (pm *BasicPeerManager) BanPeer(peer p2p.PeerID) {
-	pm.BlockPeer(peer)
+	panic("unimplemented")
 }
```

Comment:
we need to track this down

Reply:
This is in the gater commit, we don't need it.